overview.tt

```diff
@@ -0,0 +1,279 @@
+                   the way it is works?
+        -->
+        <li>complete dependencies</li>
+        <li>build sandbox</li>
+      </ul>
+      <div>
+        <p>
+          Under Nix, a build process will only find resources that have been
+          declared explicitly as dependencies. There is no way it can build
+          until everything it needs has been correctly declared. If it builds,
+          you will know you've provided a complete declaration.
+        </p>
+        <p>
+          The same build process specification is than used to provide
+          development environment. With successful build, we can guarantee that
+          everything needed to build the project is available. Even more,
+          additional development tools can be added to assist the development.
+        </p>
+        <p>
+          Long are gone the days developers saying:
+          <strong>But it works on my machine!</strong>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <!-- collaboration, sharing -->
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        <!-- TODO: missing the emotional part
+        -->
+        One tool, many languages.
+      </h2>
+      <p>
+        With today's polyglot development environments, each language has its
+        own way of building and testing. Encourage cross-team development by
+        providing a single way of building everything.
+      </p>
+      <ul>
+        <li>language agnostic</li>
+        <li>portable</li>
+      </ul>
+      <div>
+        <p>
+          Keeping the project's installation instructions up to date is a never
+          ending task. Nix allows you to specify your system dependencies
+          alongside your project's dependencies in a declarative way.
+        </p>
+        <p>
+          Instead of complicated installation instructions with Nix the
+          development environment is always only one command away. And that is
+          without using any Docker or other vitalization.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-looking.svg" %]
+    <div>
+      <h2>
+        Either it works or fails always in the same way.
+      </h2>
+      <p>
+        With Nix there are two results: the code either works or fails always
+        in the same way. This gives you the power to reproduce (and avoid) any
+        build related bugs .
+      </p>
+      <ul>
+        <li>reproducibility</li>
+        <li>transparent source binary model</li>
+        <li>nix language</li>
+        <li>continuous integration</li>
+      </ul>
+      <div>
+        <p>
+          Trying to keep system dependencies of development environments in
+          sync with continuous integration (CI) is a disaster waiting to
+          happen. Being able to use the same environment locally or anywhere
+          else is putting a developer back in control.
```

Comment:
"is putting a developer back in control" -> "ensures consistent results" or something like that? Putting the developer in control is kind of vague. Or maybe we could say something like: the developer no longer needs to ask an admin to install something on the CI servers.